packages/CodingStyle/tests/Rector/Use_/RemoveUnusedAliasRector/Fixture/skip_vich_annotation.php.inc

```diff
@@ -24,34 +24,3 @@ class Entity
      */
     private $slug;
 }
-
-?>
------
-<?php
-
-namespace App\Entity;
-
-use Doctrine\ORM\Mapping as ORM;
-use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
-use Symfony\Component\Validator\Constraints as Assert;
-use Vich\UploaderBundle\Mapping\Annotation as Vich;
-
-/**
- * @ORM\Table()
- * @ORM\Entity()
- * @UniqueEntity("slug")
- * @Vich\Uploadable
- */
-class Entity
-{
-    /**
-     * @var string
-     *
-     * @ORM\Column(name="slug", type="string", length=50, unique=true)
-     * @Assert\NotBlank()
-     * @Assert\Length(max="50")
-     */
-    private $slug;
-}
-
-?>
```

Comment:
Need to rebase on `master` to prevent this

Reply:
This is how: https://help.github.com/en/articles/syncing-a-fork

I have original repo as `upstream` and then shortun in bash console that does:

```
git fetch upstream && git merge upstream/master && cb $1 && git rebase master
```

Reply:
The branch is up-to-date. I've decided to simplify the fixtures files for those failing tests I've added, as both parts of the file (separated by `-----`) where identical.